README.md

```diff
@@ -354,6 +354,16 @@ Example
 ![Example usage of codeblocks with highlighting.](assets/docs/images/readme_codeblocks_example.png)
 
 
+### Hide copy clip button
+
+copy button will be shown by default. you can override by passing hideClipboard in markdown
+
+_Example_: <pre>
+bash hideClipboard
+Failed to update kubeadmControlPlane Connection timeout connecting to Kubernetes Endpoint 
+</pre>
+
+![Result](assets/docs/images/hide_copy_button.png)
```

Comment:
@nagesh161007 instead of this example, let's put a screenshot of the markdown code. Similar to how I did with the coloredLines example above

![CleanShot 2023-06-07 at 13 31 35](https://github.com/spectrocloud/librarium/assets/29551334/b8f066fa-0bcc-49c9-84be-40d13737ca9e)